cmd/kubeadm/app/cmd/token.go

```diff
@@ -158,11 +158,11 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	tokenCmd.AddCommand(listCmd)
 
 	deleteCmd := &cobra.Command{
-		Use:                   "delete [token-value]",
+		Use:                   "delete [token-value] ...",
 		DisableFlagsInUseLine: true,
 		Short:                 "Delete bootstrap tokens on the server.",
 		Long: dedent.Dedent(`
-			This command will delete a given bootstrap token for you.
+			This command will delete a given bootstrap tokens for you.
```

Comment:
minor typo: `a` with plural `tokens`.

possibly:
`This command will delete a list of bootstrap tokens`

Reply:
done, thanks.